pizzeria/gallery.js

```diff
@@ -0,0 +1,18 @@
+// Gallery events
+
+let imgEvent = document.querySelectorAll('.img_container img');
+imgEvent.forEach(myFunction);
+
+let chenchClass = true;
+function myFunction(item) {
+    item.addEventListener('click', () => {
+        imgEvent.forEach(clearFunction);
+        if (chenchClass) {
+            item.classList.add("img_event_js");
+            chenchClass = !chenchClass;
```

Comment:
if you have common code for if and else you can move it outside of if statement:

```
if (chenchClass) {
     item.classList.add("img_event_js");
}
chenchClass = !chenchClass;
```

because here you have the same line for both cases, so try to not duplicate the code